realm-jni/src/io_realm_internal_Version.cpp

```diff
@@ -19,16 +19,16 @@
 
 #include "util.hpp"
 #include "io_realm_internal_Version.h"
-#include <tightdb/version.hpp>
+#include <realm/version.hpp>
 
-static int tightdb_jni_version = 23;
+static int realm_jni_version = 23;
```

Comment:
It was used when linking dynamically to ensure the API was matching. Not used in Android, but may be in Java at a point. Not a problem to just keep it.